examples/webcam/webcam.py

```diff
@@ -108,6 +118,15 @@ async def on_shutdown(app):
         "--port", type=int, default=8080, help="Port for HTTP server (default: 8080)"
     )
     parser.add_argument("--verbose", "-v", action="count")
+    parser.add_argument(
+        "--preferred-codec", help="Preferred codec to use (e.g. video/H264)"
```

Comment:
This should probably be --video-codec?

Reply:
I believe that `--preferred-video-codec` is better. Do you agree?

Reply:
Quoting from https://developer.mozilla.org/en-US/docs/Web/API/RTCRtpTransceiver/setCodecPreferences :

"Any codecs not included in codecs will not be considered during the process of negotiating a connection. This lets you prevent the use of codecs you don't wish to use."

.. so no, we are not setting a preference here, we are saying "it's this codec or nothing". This is a good thing too, because the codec is not negotiable if you're playing pre-encoded data. In fact this should probably not even be an option, if you force pre-encoded media we should detect the right codec (we can keep things simple and only support H.264, and bomb if the video track uses any other codec). I don't think we should support audio at all, because it's pretty much guaranteed to fail since our "generic" `pack` method does nothing to ensure we are sending packets which fit in UDP datagrams.

Reply:
I'll make this adjust. However, I will create new PRs to adjust this feature with each audio/video codec supported by aiortc.

Reply:
> if this option is set, assert that the MediaPlayer only returns a video track containing H.264, and NO audio track
Why that is necessary to return NO audio? In this case I can remove this snippet:
```
        if isinstance(frame, AudioFrame) and audio_track:
            for frame in audio_resampler.resample(frame):
                # fix timestamps
                frame.pts = audio_samples
                frame.time_base = audio_time_base
                audio_samples += frame.samples

                frame_time = frame.time
                asyncio.run_coroutine_threadsafe(audio_track._queue.put(frame), loop)
```